lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1437,6 +1438,8 @@ export class StandardController implements IController {
     public getNativeAccountId(
         request: RedirectRequest | PopupRequest | SsoSilentRequest
     ): string {
+        // Block token acquisition before initialize has been called
+        BrowserUtils.blockCallsBeforeInitialize(this.initialized);
```

Comment:
`getNativeAccountId` shouldn't be public so this isn't needed here

Reply:
Are we planning on changing this method signature? Or are we waiting for v4 to not break the surface API?

Reply:
We should change it in v3, it's not public surface - I'm fairly certain this got changed during the move to the StandardController. If it's needed in 1P let's make it protected